ext/pcntl/pcntl.stub.php

```diff
@@ -1003,3 +1003,18 @@ function pcntl_setcpuaffinity(?int $process_id = null, array $cpu_ids = []): boo
 #ifdef HAVE_SCHED_GETCPU
 function pcntl_getcpu(): int {}
 #endif
+
+#ifdef HAVE_PTHREAD_SET_QOS_CLASS_SELF_NP
+enum QosClass: int
+{
+	case UserInteractive = 0x21;
+	case UserInitiated = 0x19;
+	case Default = 0x15;
+	case Utility = 0x11;
+	case Background = 0x09;
+	case Unspecified = 0x00;
+}
```

Comment:
I don't think it should be a backed enum, exposing those values to userland is not really helpful IMHO. Also what is this Unspecified case about?